dev_guide/service_accounts.adoc

```diff
@@ -213,6 +213,38 @@ Tokens:            	robot-token-8bhpp
 ----
 ====
 
+[[checking-if-users-can-create-pods]]
+== Checking if Users Can Create Pods
+
+Using the `scc-review` and `scc-subject-review` options, you can see if an
+individual user, or a user under a specific service account, can create a pod.
+
+Using the `scc-review` option, you can check if a service account can create a
+pod. The command outputs the security context constraints that admit the
+resource.
+
+For example, to check if a user with the
+`system:serviceaccount:projectname:default` service account can a create a pod:
```

Comment:
I was asking about having "a" article before "create" verb. If I would write this, I'll write "can create a pod".